Corona-Warn-App/src/main/java/de/rki/coronawarnapp/appconfig/AppConfigModule.kt

```diff
@@ -73,6 +74,9 @@ class AppConfigModule {
     @Provides
     fun riskMapper(mapper: RiskCalculationConfigMapper): RiskCalculationConfig.Mapper = mapper
 
+    @Provides
+    fun windowRiskMapper(mapper: ExposureWindowRiskLevelConfigMapper): ExposureWindowRiskLevelConfig.Mapper = mapper
```

Comment:
Don't be lazy and shorten names that you will rarely every type: this is not a mapper for "window risk" but the `exposureWindowRiskCalculationConfigMapper`. You can maybe drop the `exposureWindow` prefix, but would suggest to leave the rest to be precise.

Reply:
i oriented this to the naming sceme of the rest of the providers in the file, i.e. the `riskMapper` above it. As this is only a `Dagger Proivder` it technically is never called from our code but only by generated code during the build.